src/bridge/IrcBridge.ts

```diff
@@ -1202,6 +1202,18 @@ export class IrcBridge {
             "Sending IRC message in %s as %s (connected=%s)",
             ircRoom.channel, bridgedClient.nick, Boolean(bridgedClient.status === BridgedClientStatus.CONNECTED)
         );
+        if (action.text.startsWith("!irc nick")) {
+            const newNick = action.text.substr(10)
+            log.info("Matrix user wants to change nick from %s to %s", bridgedClient.nick, newNick)
+            try {
+               return bridgedClient.changeNick(newNick, true)
```

Comment:
I wonder if we should await this one, and give the user some feedback if they couldn't change their nick?

Reply:
I suck at JavaScript and failed to do this. Added a todo comment.